src/common/services/request/query/fetch-plays.js

```diff
@@ -36,6 +36,32 @@ export function FetchPlaysByID(id) {
   return payload;
 }
 
+export function FetchPlaysByNameAndUser(playname, username) {
+  const env = process.env.NODE_ENV === "development";
```

Comment:
I suggest deleting this variable as it's not currently used,

```suggestion
```

Reply:
done

Reply:
@6km please check it once again